components/Modal.tsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import styles from '../styles/modal.module.scss';
 
-interface ModalProps {
+export interface BaseModalProps {
   title: string;
-  show: boolean;
   onCancel: () => void;
   onConfirm: () => void;
   showFooter?: boolean;
   children: React.ReactNode;
 }
 
+interface ModalProps extends BaseModalProps {
+  show: boolean;
```

Comment:
if the hook is controlling the show, why cant we just have it there?

Reply:
u rite, u rite